lock-api/src/main/java/com/palantir/lock/client/RemoteTimelockServiceAdapter.java

```diff
@@ -121,6 +130,28 @@ public static RemoteTimelockServiceAdapter create(
                 minLeasedTimestampGetter);
     }
 
+    public static RemoteTimelockServiceAdapter create(
+            Namespace namespace,
+            TimelockRpcClient baseRpcClient,
+            ConjureTimelockService baseConjureClient,
+            MultiClientConjureTimelockServiceBlocking multiClientConjureClient) {
```

Comment:
I am trying to scope down the parameters provided here so we can later make some classes private (e.g. `NamespacedConjureTImelockServiceImpl`)

And, all users who were broken effectively do this